### PR TITLE
November Model Batch

### PR DESCRIPTION
**Pull request by Unit1208**
Removes old models and adds new XL models, mostly for Anime- and Furry-styled generations.

### CODE REVIEW

File: stable_diffusion.json [hunk 1]
```diff
@@ -6871,5 +6631,129 @@
...
+                {
+                    "path": "mhxlAventisHorizon_v30.safetensors",
+                    "sha256sum": "BC505CAAB21724E8F4CDEC6AF7A11275BCFC1814AD814806B107474A09B5D7F3"
+                }
+            ],
+            "download": [
+                {
+                    "file_name": "mhxlAventisHorizon_v30.safetensors",
+                    "file_path": "",
+                    "file_url": "https://civitai.com/api/download/models/1026052?type=Model&format=SafeTensor&size=pruned&fp=fp16"
```
Comment (tazlin) [inline stable_diffusion.json:6660]: This does not appear to be the correct link for this model entry; it downloads 'noobevo'. Did you mean https://civitai.com/api/download/models/461878?type=Model&format=SafeTensor&size=pruned&fp=fp16?

**Review by tazlin — CHANGES_REQUESTED** [review]
Once the link/entry is fixed, this should be OK to squash and merge.

File: stable_diffusion.json (hunk 1 above)
Comment (Unit1208) [reply stable_diffusion.json]: Thanks for the catch. Yes, I did

**Review by tazlin — APPROVED** [review]

**Comment by tazlin** [comment]
Models added from baaa9fbf...baaa9fbf:
  + MHXL - Aventis Horizon v3 (stable_diffusion_xl)
    Generalist model, focusing on semi-realistic yet stylized generations.
    nsfw: True
    https://civitai.com/models/331672/mhxl-aventis-horizon-sdxl
    None

  + Nova Furry Pony v6.0 (stable_diffusion_xl)
    Stylized 2d/2.5d furry/anthro model, emphasis on detailed fur, scales, feathers, etc.
    nsfw: True
    https://civitai.com/models/503815?modelVersionId=1032006
    None

  + Nova Anime XL v8.0 (stable_diffusion_xl)
    Anime 2d/2.5d model, emphasis on detailed fur, scales, feathers, etc.
    nsfw: True
    https://civitai.com/models/376130?modelVersionId=1065689
    None

  + White Pony Diffusion 4 v4 (stable_diffusion_xl)
    Realistic Pony model with an incredibly strong emphasis on Asian people.
    nsfw: True
    https://civitai.com/models/658304?modelVersionId=1018371
    None

Models removed:
  - Zack3D
  - OpenJourney Diffusion
  - PortraitPlus
  - MoistMix
  - Elldreth's Lucid Mix
  - Protogen Infinity

**Pull request merged**